crates/net/network/src/config.rs

```diff
@@ -107,6 +107,10 @@ pub struct NetworkConfigBuilder<C> {
     discovery_v4_builder: Discv4ConfigBuilder,
     /// All boot nodes to start network discovery with.
     boot_nodes: Vec<NodeRecord>,
+    /// Nodes we always want to connect to.
+    trusted_nodes: Vec<NodeRecord>,
+    /// If we should connect to preferred nodes only.
+    trusted_only: bool,
```

Comment:
I'd like this to be called `connect_trusted_nodes_only` or similar, should be obvious that this is about connections

Reply:
Should we change it only here, or also in the implementation?

Reply:
everywhere it is used, including function names. there are a lot of peer-related settings and logic, so it's worth being explicit here